airlock_processor/shared_code/blob_operations.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+import datetime
+from azure.storage.blob import ContainerSasPermissions, generate_container_sas, BlobServiceClient
+
+from exceptions.AirlockInvalidContainerException import AirlockInvalidContainerException
+
+
+def copy_data(source_account_name: str, source_account_key: str, sa_source_connection_string: str, sa_dest_connection_string: str, request_id: str):
+    container_name = request_id
+
+    # token geneation with expiry of 1 hour. since its not shared, we can leave it to expire (no need to track/delete)
+    sas_token = generate_container_sas(account_name=source_account_name,
```

Comment:
A SAS is usually used for operations without direct access to the storage, which we have in this case, right?
Isn't there a way to copy the blob without using it?

Reply:
Added a new issue to fix this (if possible). I tried to remove it (simple approach) and it failed